WordPress/Tests/WP/CapabilitiesUnitTest.inc

```diff
@@ -0,0 +1,88 @@
+<?php
+add_posts_page( 'page_title', 'menu_title', 'admin' . 'istrator', 'menu_slug', 'function' ); // Warning.
+add_posts_page( 'page_title', 'menu_title', 'administrator', 'menu_slug', 'function' ); // Error.
+add_media_page( 'page_title', 'menu_title', 'editor', 'menu_slug', 'function' ); // Error.
+add_pages_page( 'page_title', 'menu_title', 'author', 'menu_slug', 'function' ); // Error.
+add_comments_page( 'page_title', 'menu_title', 'contributor', 'menu_slug', 'function' ); // Error.
+add_theme_page( 'page_title', $menu_title, 'subscriber', 'menu_slug', 'function' ); // Error.
+add_plugins_page( 'page_title', 'menu_title', 'super_admin', 'menu_slug', 'function' ); // Error.
+add_users_page( 'page_title', 'menu_title', 'administrator', 'menu_slug', 'function' ); // Error.
+add_management_page( 'page_title', 'menu_title', 'editor', 'menu_slug', 'function' ); // Error.
+add_options_page( 'page_title', 'menu_title', 'level_10', 'menu_slug', 'function' ); // Error.
+if ( current_user_can( 'super_admin' ) ) { } // Error.
+if( current_user_can_for_blog( '1', 'editor' ) ) { }  // Error.
+add_dashboard_page(
+	'page_title',
+	'menu_title',
+	'super_admin' /* Error */, // Error.
```

Comment:
```suggestion
	'super_admin' /* Comment */, // Error.
```

The `error` here is confusing as the `// Error` indicates the line should result in an error already. What is actually being checked is that a comment is ignored/disregarded, so let's just call it `/* Comment */`.